scripts/apps/search/components/HighlightsList.jsx

```diff
@@ -41,10 +43,17 @@ export class HighlightsList extends React.Component {
         this.timeout = $timeout.cancel(this.timeout);
     }
 
+    closeMenu() {
+        this.props.onClose && this.props.onClose(false);
```

Comment:
Instead of doing this, just set a default prop for it and do:
```js
this.props.onClose(false);
```
and further down:
```js
HighllightsList.defaultProps = {
    onClose: () => {}
}
```